samples/export.ts

```diff
@@ -0,0 +1,28 @@
+export const numberRegexp : string;
+
+export interface StringValidator {
+    isAcceptable(s: string): boolean;
+}
+
+export namespace Hoge {
+    export class Fuga {
+        name: string
+    }
+}
+
+declare namespace PIXI {
+    const VERSION: string;
+}
+
+declare module "pixi.js" {
+    export = PIXI;
+}
+
+
+
+
+
+// declare let $: JQuery;
+// export default $;
+
+
```

Comment:
Hum ... Those blank lines and comments shouldn't be here.

Reply:
Fixed in https://github.com/sjrd/scala-js-ts-importer/pull/54/commits/b661ef794dd6b4c726e9856f27e6c8d3a9b4694c